environment.yml

```diff
@@ -11,12 +11,15 @@ dependencies:
   - setuptools<60.0
   - cython
   - compilers
+  - meson
+  - ninja
   - numpy
   - openblas
+  - pkg-config  # note: not available on Windows
   - libblas=*=*openblas  # helps avoid pulling in MKL
   - pybind11
   - pythran>=0.9.12
-  - conda-build  # to allow `conda develop .`
+  - conda-build
```

Comment:
Then should we just remove this deps?

Reply:
It was useful for something, just can't quite remember what.

Reply:
I think it is really just to do `conda develop`

Reply:
I can't find it (not a searchable string), so I'll remove - we may get reminded about it later:)